Office 365/o365/ingest/parser.yml

```
@@ -321,6 +319,20 @@ stages:
           office365.investigation.alert.correlation_key: '{{parse_data.ParsedData.CorrelationKey}}'
           office365.investigation.alert.category: '{{parse_data.ParsedData.Category}}'
           office365.investigation.alert.source_type: '{{parse_data.ParsedData.SourceAlertType}}'
+          event.start: '{{json_event.message.StartTimeUtc}}'
+          event.end: '{{json_event.message.EndTimeUtc}}'
           host.name: '{{parse_data.ParsedData.MachineName}}'
+          log.level: '{{parse_data.ParsedData.Severity}}'
+          rule.name: '{{parse_data.ParsedData.AlertDisplayName}}'
+
       - set:
           event.kind: '{% if parse_data.ParsedData.IsIncident %}{{"alert"}}{% else %}{{"event"}}{% endif %}'
+
+          ## Here is a problem with types, because all of this fields should be lists but in our definition they are strings
+          event.url: '{% for Property in parse_data.ParsedData.Entities %} {{Property.Urls}} {% endfor %}'
+#          source.ip: '{% for Property in  parse_data.ParsedData.Entities %} {{Property.SenderIP}} {% endfor %}'
+          email.subject: '{% for Property in parse_data.ParsedData.Entities %} {{Property.Subject}} {% endfor %}'
+          source.domain: '{% for Property in parse_data.ParsedData.Entities %} {{Property.P1SenderDomain}} {% endfor %}'
+          threat.enrichments: '{% for Property in parse_data.ParsedData.Entities %} {{Property.Threats}} {% endfor %}'
+          email.sender.address: '{% for Property in parse_data.ParsedData.Entities %} {{Property.Sender}} {% endfor %}'
+          destination.user.email: '{% for Property in parse_data.ParsedData.Entities %} {{Property.Recipient}} {% endfor %}'
```

[`email.to.address`](https://www.elastic.co/guide/en/ecs/current/ecs-email.html#field-email-to-address) seems more suitable. Transform as an array as expected by the field
```suggestion
          email.to.address: '[{% for Property in parse_data.ParsedData.Entities %} "{{Property.Recipient}}", {% endfor %}]'
```